modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderImport.testcase

```diff
@@ -197,6 +197,138 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-9373 Verify pagination is working properly in preview when importing a Order from Placed Orders"
+	@priority = 5
+	test CanChangePaginationInPreviewWhenImportOrderFromPlacedOrders {
+		property portal.acceptance = "true";
+
+		task ("Given the Money Order payment is activated") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+
+			CommerceNavigator.gotoPortlet(
+				category = "Store Management",
+				portlet = "Channels");
+
+			CommerceNavigator.gotoTableListEntry(entryName = "Minium Portal");
+
+			CommerceChannels.activatePaymentMethod(paymentMethod = "Money Order");
+		}
+
+		task ("And a completed order with multiple products is placed") {
+			var orderItems = "MIN93015:1,MIN93017:1,MIN93027:1,MIN93018:1,MIN93019:1,MIN93020:1,MIN93021:1,MIN93023:1,MIN00673:1,MIN00675:1,MIN55859:1,MIN55860:1,MIN55861:1";
+
+			CommerceJSONOrdersAPI._addCommerceOrder(
+				accountEntryName = "Commerce Account",
+				orderItems = ${orderItems},
+				orderStatus = 1,
+				paymentMethod = "money-order",
+				paymentStatus = 2);
+		}
+
+		task ("And the buyer is importing the order from Orders") {
+			User.logoutPG();
+
+			CommerceLogin.miniumLogin(
+				password = "test",
+				urlAppend = "web/minium",
+				userEmailAddress = "buyer@liferay.com");
+
+			CommerceAcceleratorsInitializer.createNewOrderFromPendingOrders();
+
+			CommerceOrders.importOrderFromType(orderType = "Orders");
+
+			SelectFrameTop();
+
+			IFrame.selectModalFrame();
+
+			Button.click(button = "Select");
+		}
+
+		task ("And can see data set is filled with the following data in the preview modal and only four items are displayed") {
+			Pagination.changePagination(itemsPerPage = "4 items");
+
+			CommerceEntry.viewPaginationResults(results = "Showing 1 to 4 of 13");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "ABS Sensor",
+				productQuantity = 1,
+				productSku = "MIN93015",
+				totalPrice = "50.00",
+				unitPrice = "50.00");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Premium Brake Fluid",
+				productQuantity = 1,
+				productSku = "MIN93017",
+				totalPrice = "100.00",
+				unitPrice = "100.00");
+
+			CommerceOrders.assertOrderImportPreviewEntry(
+				importStatus = "OK",
+				productName = "Wear Sensors",
+				productQuantity = 1,
+				productSku = "MIN93027",
+				totalPrice = "60.00",
+				unitPrice = "60.00");
```

Comment:
@Brunno0liveira I don't think we need to assert all these details for this test. I believe the sku/name is enough.
This applies to all these assertions in the 3 tests